torch/_inductor/codegen/cpp.py

```diff
@@ -2902,7 +2892,7 @@ def load(self, name: str, index: sympy.Expr):
                 self.disable_vec("not a loop")
                 return var
 
-            if load_dtype not in self.load_supported_dtypes and (
+            if load_dtype not in self.supported_dtypes and (
```

Comment:
Also check things in `constant` and `index_expr`?

Reply:
Unify the list in `constant`, `to_dtype`. I didn't see the dtype list in `index_expr`.

Reply:
It is a typo.